contributors.md

```diff
@@ -300,5 +482,44 @@ OSCAR project.
     {% endif %}
  </li>
 {% endif %}
+{% endif %}
+{% endfor %}
+</ul>
+
+
+## Former Contributors
+
+The following people did contribute to the OSCAR project in the past, but have not added code since June 2023.
```

Comment:
```suggestion
The following people did contribute to the OSCAR project in the past, but have not contributed code changes in the past 12 month (as of June 2024).
```
otherwise this will be just factually wrong in short order.


Also, I think we should also somewhere place instructions how to update these lists, i.e. mention running `git shortlog -s --since="1 year ago"` in all our repositories. 

And finally we kinda need to remember to update this regularly. Someone should set a regular calendar reminder maybe? Or even partially automate it via a GitHub action (might be too much work for too little gain, though)

Reply:
I wanted to avoid this issue, hence the hard-coded date. Honestly, I believe we do not want to repeat this exercise once per month or so. Let me ping @aaruni96 so we can gauge if it is feasible to automate the process. If not, I would suggest to keep the hard-coded date. At least for the next month or so.

Reply:
It would definitely be possible to automate this, but not worth the effort.

But as I see it, your current text does not avoid "the issue", to the contrary: if we don't update the page regularly, then in 6 months it'll suggest that "the following people" did not contribute for the past *18* month. Moreover, if one of them actually makes a contribution next week, we still make this (now incorrect) claim about them.

While with my version, it is clear that (a) this is about "contribution of the past 12 month" (so the next time someone wants to update it, they know what the cut-off point is), and it also makes it clearly that the list is current as of June 2024 -- so someone reading it in December can take this into account "ah, so that's why I am listed under 'former contributor' -- guess I should ping them to update this page"

Reply:
I believe we might be talking past each other. Still,

* Let me add your suggestion.
* I will also add a text at the bottom of the page, which says "Last updated: ...".

Then it should be clear what time frame was considered when creating this list.

Reply:
I'm sure I can come up with some convoluted way to automatically update this, but its not going to be quickly.

Its trivial to ask for "date one year ago" (`date -d "last year" +"%Y-%m-%d"`), but less clear what to do with the the output of git shortlog, and how to programmatically modify an md file.

Reply:
Let us gauge in a month or two (presumably someone - maybe I) has to repeat this on a regular (monthly?) basis. If this process turns out to be time consuming each time, then we can investigate further.